ibtk/include/ibtk/FEDataManager.h

```diff
@@ -548,14 +548,43 @@ class FEDataManager : public SAMRAI::tbox::Serializable
      */
     std::map<std::string, std::unique_ptr<libMesh::PetscVector<double> > > d_L2_proj_matrix_diag_ghost;
 
+private:
+    /// TODO: Remove this member variable whenever COORDINATES_SYSTEM_NAME is removed.
+    std::string& d_coordinates_system_name;
+
 public:
     /*!
      * \brief The name of the equation system which stores the spatial position
      * data. The actual string is stored by FEData.
      *
      * \note The default value for this string is "coordinates system".
      */
-    std::string& COORDINATES_SYSTEM_NAME;
+    [[deprecated(
```

Comment:
These annotations are, I think, C++17: we'll need to either update to C++17 or define a compatibility macro `IBAMR_DEPRECATED("message")`

Reply:
Let's do the latter...